params/config.go

```diff
@@ -137,12 +137,17 @@ var (
 	}
 )
 
-// VMLogTarget sets the output target of vmlog.
-// The values below can be OR'ed.
-//  - 0x0: no output (default)
-//  - 0x1: file (DATADIR/logs/vm.log)
-//  - 0x2: stdout (like logger.DEBUG)
-var VMLogTarget = 0x0
+var (
+	// VMLogTarget sets the output target of vmlog.
+	// The values below can be OR'ed.
+	//  - 0x0: no output (default)
+	//  - 0x1: file (DATADIR/logs/vm.log)
+	//  - 0x2: stdout (like logger.DEBUG)
+	VMLogTarget = 0x0
+
+	// TODO-klaytn temporal number for test
+	KIP71CompatibleBlockNum = big.NewInt(95000000)
```

Comment:
Time to remove this

Reply:
We can fix the block number now? I think we can fix this after the hardforked block number is confirmed.